frontend/public/components/app.jsx

```diff
@@ -135,9 +135,27 @@ class App extends React.PureComponent {
     analyticsSvc.route(pathname);
   }
 
+  getProductName () {
+    switch (window.SERVER_FLAGS.logoImageName) {
+      case 'origin':
+        return 'OpenShift Origin';
+      case 'ocp':
+        return 'OpenShift Container Platform';
+      case 'online':
+        return 'OpenShift Online';
+      case 'tectonic':
+        return 'Tectonic';
+      default:
+        return null;
```

Comment:
@alecmerdler Would you object to making the default branding be OpenShift in the master branch and setting `BRIDGE_LOGO_IMAGE_NAME=tectonic` for Tectonic installs?

Reply:
I would prefer the default make sense when running against pure upstream Kubernetes, but I think we just fallback to Tectonic logo currently.

Reply:
Fair, but yeah it currently defaults to Tectonic, which isn't neutral either

Reply:
@spadgett Best option would be to not render the top bar if running on pure upstream Kubernetes.

Reply:
It's more than that. There are favicons, doc links, and branding in other parts of the console.